postprocessing/janus-pp-rec.c

```diff
@@ -439,20 +442,23 @@ int main(int argc, char *argv[])
 			offset += 2;
 			if(len > 0 && !parsed_header) {
 				/* This is the info header */
-				bytes = fread(prebuffer, sizeof(char), len, file);
+				char *json_header = g_new(char, len+1);
```

Comment:
I don't get the purpose of using `json_header` (heap allocation) in place of `prebuffer` (stack).
By using `json_header` freeing must be taken into account, thus potentially leading to future bugs.
Both `char *` will be NULL terminated, so I don't really get the point of this. Am I missing something?

Reply:
See here: https://github.com/meetecho/janus-gateway/pull/2422#discussion_r517671882 (tl;dr: prebuffer's 1500 char size might not be enough)

Reply:
I see. Thanks for clarifying then!